provider/pkg/gen/dotnet-templates/helm/Unwraps.cs

```diff
@@ -17,42 +17,52 @@
 
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Text.Json.Serialization;
 using Pulumi.Kubernetes.Yaml;
 
 namespace Pulumi.Kubernetes.Helm
 {
     internal class BaseChartArgsUnwrap
     {
+        [JsonPropertyName("api_versions")]
         public ImmutableArray<string> ApiVersions { get; set; }
         public string? Namespace { get; set; }
         public ImmutableDictionary<string, object> Values { get; set; } = null!;
         public List<TransformationAction> Transformations { get; set; } = null!;
+        [JsonPropertyName("resource_prefix")]
         public string? ResourcePrefix { get; set; }
+        [JsonPropertyName("release_name")]
+        public string ReleaseName { get; set; } = null!;
```

Comment:
@mikhailshilkov I think this is causing problems and I'm wondering if there's a better way to handle this. I need a `release_name` field to be set on the JSON input to the invoke, but it shouldn't be a user-facing field. Any thoughts?

Reply:
I keep getting the following error, which I thought might be related to this change:
```
 error: Pulumi.ResourceException: A possible object cycle was detected which is not supported. This can either be due to a cycle or if the object depth is larger than the maximum allowed depth of 0.
       at Pulumi.Kubernetes.Helm.V3.Chart.<>c__DisplayClass0_0.<.ctor>b__1(ValueTuple`2 values)
       at Pulumi.Output`1.ApplyHelperAsync[U](Task`1 dataTask, Func`2 func)
       at Pulumi.Output`1.Pulumi.IOutput.GetDataAsync()
       at Pulumi.Serialization.Serializer.SerializeAsync(String ctx, Object prop)
       at Pulumi.Deployment.SerializeFilteredPropertiesAsync(String label, IDictionary`2 args, Predicate`1 acceptKey)
       at Pulumi.Deployment.SerializeAllPropertiesAsync(String label, IDictionary`2 args)
       at Pulumi.Deployment.RegisterResourceOutputsAsync(Resource resource, Output`1 outputs)
       at Pulumi.Deployment.Runner.<>c__DisplayClass7_0.<<WhileRunningAsync>g__HandleCompletion|0>d.MoveNext()
    --- End of stack trace from previous location where exception was thrown ---
       at Pulumi.Deployment.Runner.WhileRunningAsync()
```

Reply:
Honestly, this doesn't ring a bell... I can't immediately make sense of this error message.

Reply:
Actually, this error seems to be related to Transformations. The code works as long as the Chart doesn't include any transformations.

Reply:
It turns out that I was accidentally serializing a few extra properties (including transformations) that I didn't intend to. Fixed the serialization, and it works now. 🎉